public/styleguide/html/styleguide.html

```diff
@@ -824,39 +942,77 @@ <h3 class="sg-code-head">HTML</h3>
 						<h2 class="sg-pattern-head" ><a href="../../patterns/00-atoms-text-type-styles/00-atoms-text-type-styles.html" class="patternLink" data-patternpartial="atoms-type-styles">Type Styles</a></h2>
 						<div class="sg-pattern-body">
 							<div class="spacing--half">
-  <p class="font--serif--xxl">.font--serif--xl</p>
-  <p class="font--serif--xl">.font--serif--xl</p>
-  <p class="font--serif--l">.font--serif--l</p>
-  <p class="font--serif--m">.font--serif--m</p>
-  <p class="font--serif--s">.font--serif--s</p>
-  <p class="font--serif--xs">.font--serif--xs</p>
-
-  <p class="font--sans--xxl">.font--sans--xxl</p>
-  <p class="font--sans--xl">.font--sans--xl</p>
-  <p class="font--sans--l">.font--sans--l</p>
-  <p class="font--sans--m">.font--sans--m</p>
-  <p class="font--sans--s">.font--sans--s</p>
-  <p class="font--sans--xs">.font--sans--xs</p>
+  
+
+<!-- Primary font sizes -->
+  <p class="font--primary--xl">.font--primary--xl</p>
+  <p class="font--primary--l">.font--primary--l</p>
+  <p class="font--primary--m">.font--primary--m</p>
+  <p class="font--primary--s">.font--primary--s</p>
+  <p class="font--primary--xs">.font--primary--xs</p>
+ 
+<!-- Secondary Font Sizes   -->
+  <p class="font--secondary--xl">.font--secondary--xl</p>
+  <p class="font--secondary--l">.font--secondary--l</p>
+  <p class="font--secondary--m">.font--secondary--m</p>
+  <p class="font--secondary--s">.font--secondary--s</p>
+  <p class="font--secondary--xs">.font--secondary--xs</p>
+
+<!-- Tertiary Font Sizes -->
+  <p class="font--tertiary--xl">.font--tertiary--xl</p>
+  <p class="font--tertiary--l">.font--tertiary--l</p>
+  <p class="font--tertiary--m">.font--tertiary--m</p>
+
+<!-- Font Weights -->
+  <p class="font--primary--m fw--800">.font--weight--800</p>
+  <p class="font--primary--m fw--600">.font--weight--600</p>
+  <p class="font--primary--m fw--400">.font--weight--400</p>
```

Comment:
@SRHannah18 I don't think these are necessary. I don't think we have a `.font--weight--800, 600, or 400` class.